3rdparty/CMakeLists.txt

```diff
@@ -3,20 +3,20 @@ execute_process(COMMAND git submodule update --init --recursive
 )
 
 
-option(SPIRV_HEADERS_SKIP_EXAMPLES "SPIRV-Headers examples" ON)
+set(SPIRV_HEADERS_SKIP_EXAMPLES ON CACHE BOOL "Skip SPIRV-Headers examples" FORCE)
```

Comment:
A. Why are you using `FORCE` all over the place?
B. All of the subfolders specify the things with `option` not `set`, why are you using `set` ?

Reply:
here: https://github.com/buildaworldnet/IrrlichtBAW/issues/264

Reply:
Using `FORCE` so CMake overwrite current settings and people don't need to clear build folder and run CMake again. A minor issue though.